// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";


message t_crc_stats {
   required uint32 du_ue_index = 1;
   required uint32 cons_max = 2;
   required uint32 succ_tx = 3;
   required uint32 cnt_tx = 4;
   repeated uint32 retx_hist = 5;
   required uint32 harq_failure = 6;
   required uint32 min_sinr = 7;
   required int32 min_rsrp = 8;
   required uint32 max_sinr = 9;
   required int32 max_rsrp = 10;
   required uint32 sum_sinr = 11;
   required int32 sum_rsrp = 12;
   required uint32 cnt_sinr = 13;
   required uint32 cnt_rsrp = 14;
}

message crc_stats {
   required uint64 timestamp = 1;
   repeated t_crc_stats stats = 2;
}




